### ISSUE TITLE
Importing .evnt files fails with AttributeError

### ISSUE THREAD

**Issue by tutapitk** [JYU-IBA/potku#59]
# Description

Importing an .evnt file fails with AttributeError:
```
Traceback (most recent call last):
    File "dialogs/measurement/import_measurement.py", line 193, in __import_files
AttributeError: 'Measurement' object has no attribute 'directory_data'
```

# Cause

`Measurement.directory_data` was refactored to `Measurement.get_data_dir()`. However, not all places in code were updated.

# Resolution

Change all code occurrences of `Measurement.directory_data` to `Measurement.get_data_dir()`.

**Comment by tutapitk** [comment]
Fixed in df00bd9fb7dea8dd8b1c5277418294762a75c664.

Regex searching for `request\S*\.dir`, `measurement\S*\.dir`, `sample\S*\.dir`, `simulation\S*\.dir` or `\S*\.dir` did not result in other similar cases, which means that this bug was probably a one-off mistake.

**Comment by tutapitk** [comment]
On a second thought, reopening the issue because it is not merged yet.
